src/constants/blockchain.ts

```diff
@@ -72,7 +119,6 @@ export const CHAINS: Record<ChainId, ChainMetadata> = {
     name: "Polygon",
     chainId: ChainId.POLYGON,
     logoURI: polygonLogo,
-    rpcUrl: "https://polygon-rpc.com/",
```

Comment:
Why drop this RPC URL?

Reply:
was an artifact of copy/paste, see other answer